source/qb64pe.bas

```diff
@@ -12577,7 +12577,10 @@ END IF
 'Clear nm output from previous runs
 FOR x = 1 TO ResolveStaticFunctions
     IF LEN(ResolveStaticFunction_File(x)) THEN
-        s$ = MakeNMOutputFilename$(ResolveStaticFunction_File(x))
+        s$ = MakeNMOutputFilename$(ResolveStaticFunction_File(x), 0)
+        IF _FILEEXISTS(s$) THEN KILL s$
+
+        s$ = MakeNMOutputFilename$(ResolveStaticFunction_File(x), 1)
         IF _FILEEXISTS(s$) THEN KILL s$
```

Comment:
Not sure if we should do a `ClearBuffers` in the two _FILEEXISTS THEN KILL lines too, just in case the file was already loaded into memory. It would definitly not break things, as it ignores calls for unknown buffers (see utilities\s-buffer\sb_qb64pe_extension.bm).
i.e. instead of `IF _FILEEXISTS(s$) THEN KILL s$` write `IF _FILEEXISTS(s$) THEN KILL s$: ClearBuffers s$`

Reply:
Thanks for the suggestion, yeah I'll add that :)